core/components/atoms/box/Box.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { BaseProps, BaseHtmlProps } from '@/utils/types';
+import styles from '@css/components/box.module.css';
+
+export type BoxDisplay = 'inline' | 'inline-block' | 'block' | 'flex' | 'inline-flex';
+export type BoxElevation = 'flat' | 'raised' | 'overlay';
+export type BoxBorderRadius = 'none' | 'small' | 'medium' | 'large' | 'full';
+export type BoxPadding = 'none' | 'small' | 'medium' | 'large';
+export type BoxBackground =
+  | 'transparent'
+  | 'default'
+  | 'subtle'
+  | 'primary'
+  | 'secondary'
+  | 'success'
+  | 'alert'
+  | 'warning';
+export type BoxBorder = 'none' | 'default' | 'primary' | 'secondary' | 'success' | 'alert' | 'warning';
+export type BoxPosition = 'static' | 'relative' | 'absolute' | 'fixed' | 'sticky';
+export type BoxFlex = 1 | 2 | 3;
```

Comment:
Why are we accepting so many props related to css when we can easily handle all this using classnames and utility classes??